processor/worker_test.go

```diff
@@ -214,6 +214,12 @@ func (*mockWorkerHandle) stats() *processorStats {
 		DBReadThroughput: func(partition string) stats.Measurement {
 			return stats.Default.NewStat("db_read_throughput", stats.CountType)
 		},
+		StatDBReadOutOfOrder: func(partition string) stats.Measurement {
+			return stats.Default.NewStat("db_read_out_of_order", stats.CountType)
+		},
+		StatDBReadOutOfSequence: func(partition string) stats.Measurement {
+			return stats.Default.NewStat("db_read_out_of_sequence", stats.CountType)
+		},
```

Comment:
We can probably use NOP here?
```suggestion
		StatDBReadOutOfOrder: func(partition string) stats.Measurement {
			return stats.NOP.NewStat("db_read_out_of_order", stats.CountType)
		},
		StatDBReadOutOfSequence: func(partition string) stats.Measurement {
			return stats.NOP.NewStat("db_read_out_of_sequence", stats.CountType)
		},
```